### DISCUSSION TITLE
Get subfolders recursively

### DISCUSSION THREAD

**Discussion by pat721** [react-dropzone/react-dropzone#1300] [Q&A]
Hello everyone,

I'm trying to take subfolders of folders recursively on drop, but I just don't know how to handle this. 
I checked the PR #616 which was mentioned in ticket #11 but I'm not able to make sense of it. There is also a link to a site that does not longer exist in the documentation: [using-third-party-plugins](https://react-dropzone.js.org/#using-third-party-plugins).

Maybe someone can help me on how to approach this. 

TL;DR: I'm trying to drop a folder and get all contents including subfolders

**Comment by pat721** [comment]
I have worked around in my application, so I don't need this any more.
